auth/token_verifier.go

```diff
@@ -476,19 +473,20 @@ func parsePublicKey(kid string, key []byte) (*publicKey, error) {
 	return &publicKey{kid, pk}, nil
 }
 
-func findMaxAge(resp *http.Response) (*time.Duration, error) {
+func findMaxAge(resp *http.Response) *time.Duration {
+	duration := time.Duration(0) * time.Second
```

Comment:
Can this line go with the default return value at the bottom of the function?

Reply:
Thanks! Moved the default value to the bottom